DEVELOPMENT.md

```diff
@@ -78,6 +78,21 @@ When enabled by default, this also works with editor plugins like
 
 [vim-fugitive]: https://github.com/tpope/vim-fugitive
 
+### iBazel: A file watcher for Bazel.
+
+Bazel is capable incrementally builds where it builds only subset of files that
+are impacted by file changes, but it does not come with a file watcher. For an
+improved developer experience, start TensorBoard with `ibazel` instead of
+`bazel` which will automatically re-build and start the server when files
+change.
+
+If you do not have `ibazel` binary on your system, you can do use command below.
```

Comment:
Grammar tweak:
If you do not have the `ibazel` binary on your system, you can use the command below.